tests/test_server/test_lineage/test_column_lineage.py

```diff
@@ -0,0 +1,209 @@
+from http import HTTPStatus
+
+import pytest
+from httpx import AsyncClient
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from tests.fixtures.mocks import MockedUser
+from tests.test_server.utils.convert_to_json import (
+    datasets_to_json,
+    direct_column_lineage_to_json,
+    indirect_column_lineage_to_json,
+    inputs_to_json,
+    jobs_to_json,
+    operation_parents_to_json,
+    operations_to_json,
+    outputs_to_json,
+    run_parents_to_json,
+    runs_to_json,
+)
+from tests.test_server.utils.enrich import enrich_datasets, enrich_jobs, enrich_runs
+from tests.test_server.utils.lineage_result import LineageResult
+from tests.test_server.utils.merge import merge_io_by_jobs, merge_io_by_runs
+
+pytestmark = [pytest.mark.server, pytest.mark.asyncio, pytest.mark.lineage]
+
+
+async def test_get_operation_lineage_simple(
+    test_client: AsyncClient,
+    async_session: AsyncSession,
+    branchy_lineage_with_column_lineage: LineageResult,
+    mocked_user: MockedUser,
+):
+    lineage = branchy_lineage_with_column_lineage
+    operation = lineage.operations[0]
+    run = next(run for run in lineage.runs if run.id == operation.run_id)
+    job = next(job for job in lineage.jobs if job.id == run.job_id)
```

Comment:
Lineage fixture has the same relations J -> R -> O every time, you can get run and job by index.